tests/test_optimizely.py

```diff
@@ -2256,124 +2391,423 @@ def test_get_feature_variable_integer_for_feature_in_rollout(self):
         'feature_enabled': True,
         'source': 'rollout',
         'variable_key': 'count',
-        'variable_value': 399,
+        'variable_value': 399,
+        'variable_type': 'integer',
+        'source_info': {}
+      }
+    )
+
+  def test_get_feature_variable_string_for_feature_in_rollout(self):
+    """ Test that get_feature_variable_double returns Double value as expected \
```

Comment:
Comment talks about double but test is for feature variable string API.

Reply:
Updated comment to match its method.